neurons/miner.py

```diff
@@ -43,158 +43,38 @@
-            self.structured_search_engine.crawl_and_index_data(
-                query_string=query.query_string,
-                author_usernames=None,
-                # crawl and index more data than needed to ensure we have enough to rank
-                max_size=crawl_size,
-            )
-
-        ranked_docs = self.structured_search_engine.search(query)
-
-        bt.logging.debug(f"{len(ranked_docs)} ranked_docs", ranked_docs)
-        query.results = ranked_docs
-        end_time = datetime.now()
-        elapsed_time = (end_time - start_time).total_seconds()
-        bt.logging.info(
-            f"processed SearchSynapse in {elapsed_time} seconds",
-        )
-        return query
+        pass
 
     async def forward_structured_search(
         self, query: StructuredSearchSynapse
     ) -> StructuredSearchSynapse:
-
-        start_time = datetime.now()
-        bt.logging.info(
-            f"received StructuredSearchSynapse... timeout:{query.timeout}s ", query
-        )
-        self.check_version(query)
-
-        # miners may adjust this timeout config by themselves according to their own crawler speed and latency
-        if query.timeout > 12:
-            # do crawling and indexing, otherwise search from the existing index directly
-            crawl_size = max(self.config.neuron.crawl_size, query.size)
-            self.structured_search_engine.crawl_and_index_data(
-                query_string=query.query_string,
-                author_usernames=query.author_usernames,
-                # crawl and index more data than needed to ensure we have enough to rank
-                max_size=crawl_size,
-            )
-
-        # disable crawling for structured search by default
-        ranked_docs = self.structured_search_engine.search(query)
-        bt.logging.debug(f"{len(ranked_docs)} ranked_docs", ranked_docs)
-        query.results = ranked_docs
-        end_time = datetime.now()
-        elapsed_time = (end_time - start_time).total_seconds()
-        bt.logging.info(
-            f"processed StructuredSearchSynapse in {elapsed_time} seconds",
-        )
-        return query
+        pass
 
     async def forward_semantic_search(
         self, query: SemanticSearchSynapse
     ) -> SemanticSearchSynapse:
-
-        start_time = datetime.now()
-        bt.logging.info(
-            f"received SemanticSearchSynapse... timeout:{query.timeout}s ", query
-        )
-        self.check_version(query)
-
-        ranked_docs = self.structured_search_engine.vector_search(query)
-        bt.logging.debug(f"{len(ranked_docs)} ranked_docs", ranked_docs)
-        query.results = ranked_docs
-        end_time = datetime.now()
-        elapsed_time = (end_time - start_time).total_seconds()
-        bt.logging.info(
-            f"processed SemanticSearchSynapse in {elapsed_time} seconds",
-        )
-        return query
+        pass
 
     async def forward_discord_search(
```

Comment:
nit: let's add some comments to those tasks deprecated

Reply:
solved by update